app/views/user_mailer/batch_subscription_notification.html.erb

```diff
@@ -3,51 +3,50 @@
   <% @seen_summary = {} %>
   <% @creations.each_with_index do |creation, index| %>
 
-    <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-    <% work_link = style_link(this_work.title.html_safe, work_url(this_work)) %>
+<% this_work = creation.respond_to?(:work) ? creation.work : creation %>
+<% pseud = this_work.pseuds.map{|p| p.name}.to_sentence %>
+<% byline = this_work.pseuds.map{|p| p.byline}.to_sentence %>
+<% title = this_work.title.html_safe %>
+<% if creation.is_a?(Work) then
+     link  = style_link(this_work.title.html_safe,work_url(this_work))
+     word_count = this_work.word_count
+   else
+     link  = style_link(this_work.title.html_safe,work_chapter_url(this_work, creation))
+     chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe
+   end %>
+<% if creation.is_a?(Work) %>
+<%= if this_work.backdate then
+t('user_mailer.batch_subscription_notification.html.work_backdated', byline: byline, title: title, link: link, word_count: word_count ).html_safe
+else
+t('user_mailer.batch_subscription_notification.html.work_new', byline: byline, title: title, link: link, word_count: word_count).html_safe
+end %>
+<% else %>
+<%= t('user_mailer.batch_subscription_notification.html.new_chapter_of',byline: byline, title: title, link: link, word_count: creation.word_count, chapter_title: chapter_title).html_safe %>
+<%end%>
+<%= t( 'user_mailer.batch_subscription_notification.html.collected_by', pseud: this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe).html_safe %>
 
-    <%= this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe %> posted a
-    <% if creation.is_a?(Work) %>
-      <%= this_work.backdate ? "backdated" : "new" %> work:
-    <% else %>
-      new chapter of <i><b><%= work_link %></b></i> (<%= this_work.word_count %> words):
-      <% chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe %>
-    <% end %>
-
-    <p>
-      <i>
-        <%= style_bold(creation.is_a?(Work) ? work_link :
-          style_link(chapter_title, work_chapter_url(this_work, creation))) %>
-      </i> (<%= creation.word_count%> words)
-
-      <% unless @seen[this_work.id] %>
-        <br>
-
-        by <%= this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe %>
-      <% end %>
-    </p>
+<% if !creation.summary.blank? && creation.is_a?(Chapter) %>
+<%= t 'user_mailer.batch_subscription_notification.chapter_summary'%> <%= raw to_plain_text(sanitize_field(creation, :summary)) %>
```

Comment:
There's a space missing between `'user_mailer.batch_subscription_notification.chapter_summary'` and the `%>`